kratos/python_scripts/read_materials_process.py

```diff
@@ -86,7 +86,9 @@ def _AssignPropertyBlock(self, data):
                 },
                 "Variables" : {
                     "YOUNG_MODULUS" : 200e9,
-                    "POISSION_RATIO" : 0.3
+                    "POISSION_RATIO" : 0.3,
```

Comment:
DOES THIS WORK? because it is badly spelt...

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/31132258-a2f330f0-a85c-11e7-9d6b-f03585648459.png)

Reply:
Los árboles no dejan ver el bosque

Reply:
the big problem is that this should deliver an error. If it does not we have a problem

Reply:
Is an example, is commented, not computed

Reply:
It throws an error:
```
NameError: name 'POISSION_RATIO' is not defined
```